code/modules/mob/living/simple_animal/abnormality/he/silent_girl.dm

```diff
@@ -89,24 +92,29 @@
 	return
 
 /mob/living/simple_animal/hostile/abnormality/silent_girl/zero_qliphoth(mob/living/carbon/human/user)
+	var/insanity_timer = insanity_short
 	var/dead_list = guilty_people
+	var/list/potential_guilt = list()
 	for(var/mob/living/carbon/human/dead_body in dead_list)
 		if(dead_body.stat == DEAD || isnull(dead_body))
 			guilty_people -= dead_body
 	if (!LAZYLEN(guilty_people)) // No Guilty on 0 counter? Find a random person and take them <3
-		var/list/potential_guilt = list()
```

Comment:
This doesn't need to be moved and I'd like to avoid declaring variables when they may not be used. It's literally my only issue with this.

Reply:
Fixed in latest commit.